### PR TITLE
new RAxML easyconfigs

### PR DESCRIPTION
**Pull request by pescobar**
Upgraded to latest RAxML release 8.1.1

Switched to MakeCp easyblock. Easier to read easyconfig and also documentation and examples can be copied.

Added AVX easyconfigs

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 4 out of 4
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/decc1ad3436f565bdd5e for a full test report.

**Comment by fgeorgatos** [comment]
LGTM!

**Comment by boegel** [comment]
@pescobar: please merge https://github.com/pescobar/easybuild-easyconfigs/pull/35 to make sure the unit tests are OK here

File: easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mpi-avx.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+homepage = 'https://github.com/stamatak/standard-RAxML'
+description = "RAxML search algorithm for maximum likelihood based inference of phylogenetic trees."
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/stamatak/standard-RAxML/archive/']
+sources = ['v%(version)s.zip']
+
+makeopts = ' -f Makefile.AVX.MPI.gcc'
```
Comment (boegel) [inline easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mpi-avx.eb]: `buildopts` please

File: easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mpi-sse3.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+homepage = 'https://github.com/stamatak/standard-RAxML'
+description = "RAxML search algorithm for maximum likelihood based inference of phylogenetic trees."
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/stamatak/standard-RAxML/archive/']
+sources = ['v%(version)s.zip']
+
+makeopts = ' -f Makefile.SSE3.MPI.gcc'
```
Comment (boegel) [inline easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mpi-sse3.eb]: `buildopts` please

File: easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mt-avx.eb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+homepage = 'https://github.com/stamatak/standard-RAxML'
+description = "RAxML search algorithm for maximum likelihood based inference of phylogenetic trees."
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/stamatak/standard-RAxML/archive/']
+sources = ['v%(version)s.zip']
+
+makeopts = '-f Makefile.AVX.PTHREADS.gcc'
```
Comment (boegel) [inline easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mt-avx.eb]: `buildopts` please

File: easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mt-sse3.eb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+homepage = 'https://github.com/stamatak/standard-RAxML'
+description = "RAxML search algorithm for maximum likelihood based inference of phylogenetic trees."
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/stamatak/standard-RAxML/archive/']
+sources = ['v%(version)s.zip']
+
+makeopts = '-f Makefile.SSE3.PTHREADS.gcc'
```
Comment (boegel) [inline easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mt-sse3.eb]: `buildopts` please

**Comment by pescobar** [comment]
to be replaced by this one. https://github.com/hpcugent/easybuild-easyconfigs/pull/1985

closing

**Pull request closed without merging**